### ISSUE TITLE
docker: supervisord installed without any need

### ISSUE THREAD

**Issue by avikivity** [scylladb/scylladb#21955]
Since we no longer run jmx, we only have one process to supervise.

We may as well have the entry point exec it as is usual with docker images, and drop supervisord completely.

/cc @tnozicka @zimnx

**Comment by mykaul** [comment]
YES! But we've had an issue with dropping it last time we did it (by mistake?)  - need to see what it was back then.

**Comment by avikivity** [comment]
There's no point in running node_exporter on the scylladb container.

**Comment by mykaul** [comment]
@avikivity - see the discussion @ https://github.com/scylladb/scylladb/issues/21923

**Comment by tnozicka** [comment]
Apparently it was broken for a long time but the screenshot here show where it plumbs today https://github.com/scylladb/scylla-operator/pull/2279

I agree we shouldn't run it but then we need to redesign the monitoring dashboard / integration.
